src/current/_includes/releases/v24.1/v24.1.0.md

```diff
@@ -241,7 +241,8 @@ In CockroachDB Self-Hosted, all available features are free to use unless their
 
 Before [upgrading to CockroachDB v24.1]({% link v24.1/upgrade-cockroach-version.md %}), be sure to review the following backward-incompatible changes, as well as [key cluster setting changes](#v24-1-0-cluster-settings), and adjust your deployment as necessary.
 
-- TBD
+- [`AS OF SYSTEM TIME`]({% link v23.2/as-of-system-time.md %}) queries can no longer use a timestamp followed by a question mark to signify a future-time value. This was an undocumented syntax. [#116830](https://github.com/cockroachdb/cockroach/pull/116830)
+- The `sql.txn.read_committed_isolation.enabled` [cluster setting]({% link v24.1/cluster-settings.md %}) is now `true` by default. As a result, [`READ COMMITTED`]({% link v24.1/read-committed.md %}) transactions are **not** automatically upgraded to [`SERIALIZABLE`]({% link v24.1/demo-serializable.md %}), and will run as `READ COMMITTED` by default. On v23.2, refer to the [**Upgrades of SQL Transaction Isolation Level**]({% link v24.1/ui-sql-dashboard.md %}#upgrades-of-sql-transaction-isolation-level) graph in the DB Console to check whether any transaction is being upgraded from a weaker isolation level to `SERIALIZABLE`, and could therefore run differently on v24.1. [#118479](https://github.com/cockroachdb/cockroach/pull/118479)
```

Comment:
All of the info looks correct to me. Just one more higher level question -- should this also mention that READ COMMITTED is enterprise only?

I.e., if you don't have an enterprise license, then even if the setting is on, the auto-upgrade to SERIALIZABLE will still happen. (Maybe this is something better left for the detailed docs, and I also don't want to derail this PR too much. I'm fine with whatever you decide, just mentioning it in case.)

Reply:
OK that's an important point -- I think this needs to be a separate backward-incompatible note.

cc @mdlinville (this just keeps the wording of the release note):

```
- The `READ COMMITTED` isolation level now requires the cluster to have a valid enterprise license. [#120154](https://github.com/cockroachdb/cockroach/pull/120154)
```

Reply:
Done. PTAL because I added a little more detail to the end of your draft, and also added it to a new Enterprise Features section directly above the Backward-Incompatible Changes section.